src/wallet/test/walletdb_tests.cpp

```diff
@@ -48,10 +48,12 @@ BOOST_AUTO_TEST_CASE(walletdb_read_write_deadlock)
         LOCK(wallet->cs_wallet);
         auto legacy_spkm = wallet->GetOrCreateLegacyScriptPubKeyMan();
```

Comment:
nit, if you retouch: 
```suggestion
        auto legacy_spkm = wallet->GetOrCreateLegacyScriptPubKeyMan();
        auto legacy_spkm = wallet->GetOrCreateLegacyScriptPubKeyMan();
```